examples/aspnetcore-with-extensions-logging/Program.cs

```diff
@@ -0,0 +1,42 @@
+// -- Start an Elasticsearch Instance --
+
+using Elastic.Clients.Elasticsearch;
+using Elastic.Elasticsearch.Ephemeral;
+using Elasticsearch.Extensions.Logging;
+
+using var cluster = new EphemeralCluster("8.4.0");
+var client = CreateClient(cluster);
+//check if an instance is already running before starting
+if (!(await client.InfoAsync()).IsValidResponse)
+	cluster.Start(TimeSpan.FromMinutes(1));
+else Console.WriteLine("Using already running Elasticsearch instance");
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+
+builder.Host.ConfigureLogging((_, loggingBuilder) =>
+{
+	loggingBuilder.AddElasticsearch(client.Transport, log =>
```

Comment:
How would this work for AddElasticCloud ?

Reply:
Hi @HubDevUser, There is an `.AddElasticCloud()` extension method too.
Although not one that takes `Transport` is that the question?

Reply:
When i use .AddElasticeCloud(CloudId, ApiKey) it does not report anything to my elastic cloud instance. It does work when i do .AddElasticCloud(CloudId, ApiKey) in this https://github.com/elastic/ecs-dotnet/blob/main/examples/console-with-extensions-logging/Program.cs#L25 console example though.  .AddElasticSearch() does work on my local dev docker image. How can I make this example work on a cloud instance :'( ?

Reply:
managed to work it 

	loggingBuilder.AddElasticsearch();

	void configure(ElasticsearchLoggerOptions options)
	{
		options.ShipTo.ConnectionPoolType = ConnectionPoolType.Cloud;
		options.ShipTo.CloudId = "id";
		options.ShipTo.ApiKey = "key";
	}

	builder.Services.Configure((Action<ElasticsearchLoggerOptions>)configure);

Reply:
this works L

var builder = WebApplication.CreateBuilder(args);

// Add services to the container.

builder.Services.AddControllers();
builder.Services.AddEndpointsApiExplorer();

builder.Host.ConfigureLogging((_, loggingBuilder) =>
{
	loggingBuilder.AddElasticsearch();

	void configure(ElasticsearchLoggerOptions options)
	{
		options.ShipTo.ConnectionPoolType = ConnectionPoolType.Cloud;
		options.ShipTo.CloudId = "id";
		options.ShipTo.ApiKey = "key";
	}

	builder.Services.Configure((Action<ElasticsearchLoggerOptions>)configure);

});
var app = builder.Build();

app.UseAuthorization();
app.MapControllers();
app.Run();



this does not :

namespace WebApplication5
{
    public class Program
    {
        public static void Main(string[] args)
        {
            CreateHostBuilder(args).Build().Run();
        }

        public static IHostBuilder CreateHostBuilder(string[] args) =>
            Host.CreateDefaultBuilder(args)
                .ConfigureWebHostDefaults(webBuilder =>
                {
                    webBuilder.ConfigureLogging((builder) =>
                    {
                        builder.AddElasticsearch();

                        void configure(ElasticsearchLoggerOptions options)
                        {
                            options.ShipTo.ConnectionPoolType = Elastic.Ingest.ConnectionPoolType.Cloud;
                            options.ShipTo.CloudId = "id";
                            options.ShipTo.ApiKey = "key";
                        }

                        builder.Services.Configure((Action<ElasticsearchLoggerOptions>)configure);
                    })
                    .UseStartup<Startup>();
                });
    }
}